resources/dicts/conditions/pregnancy.json

```
@@ -1,6 +1,9 @@
 {
   "announcement": [
-    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits."
+    "m_c announced that {PRONOUN/m_c/subject} {VERB/m_c/are/is} expecting kits.",
+	  "m_c had been secretive for a while now and nobody could even begin to guess what about... until today, that is, when m_c announced {PRONOUN/m_c/poss} pregnancy!",
+          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```

```suggestion
          "The entire Clan could see the signs way before m_c did. Everyone urged {PRONOUN/m_c/object} to visit the medicine cat den to no avail. They just about gave up until, finally, m_c announced {PRONOUN/m_c/poss} pregnancy.",
```
I would suggest just doing a ctrl+shift+r to find and replace all the "medicine den"s with "medicine cat den"

---

i could have sworn it was medicine den, not medicine cats den. i definitely remember another pr i submitted where the inverse was requested